src/App.js

```diff
@@ -2,14 +2,21 @@ import "./App.css";
 import RestaurantView from "./pages/RestaurantView/RestaurantView";
 import HomePageComponent from "./pages/HomePageComponent/HomePageComponent";
 import OrderPage from "./pages/OrderPage/OrderPage";
-import { Routes, Route } from "react-router";
+import { Routes, Route, Navigate } from "react-router";
 import NotFoundPageComponent from "./pages/NotFoundPageComponent/NotFoundPageComponent";
 import RestaurantListPage from "./pages/RestaurantListPage/RestaurantListPage";
 import SignInPage from "./pages/SignInPage/SignInPage";
 import RestaurantAdminView from "./pages/RestaurantAdminView/RestaurantAdminView";
 import GeneralAdminView from "./pages/GeneralAdminView/GeneralAdminView";
 import { CheckoutPageComponent } from "./pages/CheckoutPageComponent/CheckoutPageComponent";
 import UserPage from "./pages/UserPage/UserPage";
+import Cookies from "universal-cookie";
+
+const Private = ({ children }) => {
+  const cookies = new Cookies();
+  const ticket = cookies.get("token");
+  return ticket ? children : <Navigate to={"/"}/>
```

Comment:
`return ticket ? children : <Navigate to={"/"}/>;`

Reply:
Resolved